blackboxopt/utils.py

```diff
@@ -3,29 +3,33 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-from typing import Dict, Optional, Sequence
+from typing import Dict, List, Optional, Sequence
 
 from blackboxopt.base import Objective
 
 
 def get_loss_vector(
     known_objectives: Sequence[Objective],
     reported_objectives: Dict[str, Optional[float]],
-) -> list:
+    none_replacement: float = float("NaN"),
+) -> List[float]:
     """Convert reported objectives into a vector of known objectives.
 
     Args:
-        known_objectives: [description]
-        reported_objectives: [description]
+        known_objectives: A sequence of the objectives that e.g. an optimizer was
```

Comment:
why `e.g.` here?
we could also add here: "Dictates the order for returned loss"

Reply:
"e.g." because they could also be known for some other reason. And I'll happily add the return loss order detail.